docs/topics/auth/default.txt

```diff
@@ -1395,6 +1395,14 @@ implementation details see :ref:`using-the-views`.
     * ``extra_context``: A dictionary of context data that will be added to the
       default context data passed to the template.
 
+    * ``reset_url_token``: Token parameter displayed in password reset URLs,
+      consist of two alphanumeric strings separated by a hyphen. Defaults to
+      ``'set-password'``.
```

Comment:
Maybe...

```
Token parameter displayed as component of password reset URLs, 
consists of two alphanumeric strings separated by a hyphen. Defaults to
``'set-password'``.
```

Note `consists` with the `s`. 

Q: `consists of two alphanumeric strings separated by a hyphen` Is that necessary? (Used in check_token?) If so `This must consists of...`

Reply:
Good catch :dart: , it is only required in our tests (see [uid_token](https://github.com/django/django/blob/8000767769ac37ec7f73eb9fcc4b3fc7399a5808/tests/auth_tests/urls.py#L81)). I will remove this sentence.